torch/csrc/autograd/autograd.cpp

```diff
@@ -0,0 +1,138 @@
+#include <torch/csrc/autograd/autograd.h>
+#include <torch/csrc/autograd/variable.h>
+
+#include <torch/csrc/autograd/edge.h>
+#include <torch/csrc/autograd/engine.h>
+#include <torch/csrc/autograd/function.h>
+
+namespace torch {
+namespace autograd {
```

Comment:
It would be very helpful to say "hey this code duplicates existing logic at X" here, as a reminder for people who are updating the code to update both places